README.md

```diff
@@ -297,6 +297,9 @@ spec:
 
 # Release notes
 
+## 5.13.1
+* Fixed the problem data provider can't handle `messageIds` request without startTimestamp
```

Comment:
note: I think we should also add information that resume IDs were provided in the request